notify-send.sh

```diff
@@ -243,7 +249,7 @@ done
 # urgency is always set
 HINTS=("$(make_hint byte urgency "$URGENCY")" "${HINTS[@]}")
 
-if [[ -z "$SUMMARY" ]] ; then
+if [[ -z "$SUMMARY" && -z "$BODY" ]] ; then
```

Comment:
I think the condition now can be as follows

    if [[ "$SUMMARY_SET" = n ]] ; then

Reply:
Let's consider the case of `notify-send.sh '' ''` with both the summary and body empty.

I would say that this is not a valid notification, on my system it is just a tiny, empty square.

For `notify-send.sh '' '' -i info` it is just the icon.  Personally I don't think this is very useful either.

If these empty notifications are something we want to allow, then your suggested change is correct, otherwise I think it would be better to print the help/usage text as this PR does currently.

Reply:
Yes, I would allow it. In Gnome Shell empty notifications are displayed just fine (if we ignore the fact that they are empty).

My logic was that if with this PR we allow empty required field (summary) we perhaps should not enforce non-emptiness of optional field (body).

With empty notification it's still possible to show an icon, actions - who knows who might find it useful under specific circumstances (and specific notifications implementation).